### DISCUSSION TITLE
[@types/umami] - use types from umami?

### DISCUSSION THREAD

**Discussion by nandi95** [DefinitelyTyped/DefinitelyTyped#70372] [Issues with a @types package]
Is there a way to use the types that are maintained at umami? any way to reference it? Or perhaps just copy paste into this shim for more accurate/documented types.
https://github.com/umami-software/umami/blob/47a6170a834c8937c042d26be3dc61bdadbaed37/src/tracker/index.d.ts

[a comment by an automated account is not shown]

**Comment by ncovercash** [comment]
We should remove the @types/umami package, then. The DefinitelyTyped version was created before umami maintained its own types.

**Reply by nandi95** [reply]
I don't think these types are exposed, this is their backend codebase
